sourcery_analytics/settings.py

```diff
@@ -1,24 +1,37 @@
+"""Models describing sourcery-analytics settings."""
 import pathlib
 
 import pydantic
 import tomli
 
 
 class ThresholdSettings(pydantic.BaseModel):
+    """Model describing the available thresholds and their defaults."""
+
     method_length: pydantic.PositiveInt = 15
     method_cyclomatic_complexity: pydantic.PositiveInt = 10
     method_cognitive_complexity: pydantic.PositiveInt = 10
     method_working_memory: pydantic.PositiveInt = 20
 
-    class Config:
-        extras_allowed = True
-
 
 class Settings(pydantic.BaseSettings):
+    """Model describing general sourcery-analytics settings and their construction."""
+
     thresholds: ThresholdSettings = ThresholdSettings()
 
     @classmethod
     def from_toml_file(cls, toml_file_path: pathlib.Path):
+        """Construct settings from a toml file.
```

Comment:
**suggestion (docstrings):** Please update the docstring for function: `from_toml_file`

Reason for update: Addition of detailed argument and return type descriptions.

Suggested new docstring:
```
"""Construct settings from a toml file.

Args:
    toml_file_path:
        Relative or fully-qualified path to a toml file containing
        sourcery-analytics settings.

Returns:
    A Settings instance.

"""
```